input/pagecontent/changes.md

```diff
@@ -1,3 +1,13 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-01
+- Expected Publication status: TBC
+- Based on FHIR version: 4.0.1
+
+This change log documents the significant updates and resolutions implemented from version 1.0.0-ballot to 1.0.0.
+
+#### Changes in this version
+- Corrected requirement for both system and code values for Location, Organization, Practitioner and PractitionerRole identifier search parameter to be mandatory for Requester [FHIR-46782](https://jira.hl7.org/browse/FHIR-46782)
```

Comment:
Maybe something like: "Updated requirement for the AU Core Requester to mandate both system and code values for identifier search parameters on Location, Organization, Practitioner and PractitionerRole [FHIR-46782](https://jira.hl7.org/browse/FHIR-46782).
This was a change request, not a TC.

Reply:
Amended, thanks.